### PR TITLE
feat: Add simple bot sdk.

### LINKED ISSUE #201: Feature: Add bot sdk.
### Checklist

- [X] I've searched for similar issues and couldn't find anything matching
- [X] I've discussed this feature request in the [OpenIMSDK Slack](https://join.slack.com/t/openimsdk/shared_invite/zt-1tmoj26uf-_FDy3dowVHBiGvLk9e5Xkg) and got positive feedback

### Is this feature request related to a problem?

❎ No

### Problem Description

Need to support bot sdk to get llm input and output.

### Solution Description

Add http handler.

### Benefits

Enhance system.

### Potential Drawbacks

_No response_

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by IRONICBo**
#### 🔍 What type of PR is this?

/kind feature

#### 👀 What this PR does / why we need it:

Support simple llm bot sdk.

#### 🅰 Which issue(s) this PR fixes:

Fixes #201

### CODE REVIEW

File: server/pkg/openim/sdk/bot/bot.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	"fmt"
+
+	"github.com/openimsdk/openkf/server/pkg/openim/client"
+	"github.com/openimsdk/openkf/server/pkg/openim/param/request"
+	"github.com/openimsdk/openkf/server/pkg/openim/param/response"
+)
+
+const (
+	// PATH_BOT_ASK ask with bot.
+	PATH_BOT_ASK = "/api/v1/ask"
```
Comment (cubxxw) [inline server/pkg/openim/sdk/bot/bot.go:27]: first letter is uppercase OR lowercase

File: server/pkg/openim/sdk/bot/bot.go [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	url := fmt.Sprintf("%s%s", host, PATH_BOT_ASK)
+
+	r := &response.BotQueryResponse{}
+	client := client.NewClient(url)
+	resp, err := client.POST(operationID, "", param)
+	if err != nil {
+		return r, err
+	}
+
+	r.Code = uint(resp["code"].(float64))
```
Comment (cubxxw) [inline server/pkg/openim/sdk/bot/bot.go:42]: You should check for the existence of a type assertion before making a safe type assertion

**Review by cubxxw — DISMISSED** [review]

File: server/pkg/openim/sdk/bot/bot.go (hunk 1 above)
Comment (IRONICBo) [reply server/pkg/openim/sdk/bot/bot.go:27]: Thanks!

File: server/pkg/openim/sdk/bot/bot.go (hunk 2 above)
Comment (IRONICBo) [reply server/pkg/openim/sdk/bot/bot.go:42]: Helpful~

**Review by cubxxw — APPROVED** [review]
lgtm

**Pull request merged**